<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QDialog" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bienvenue </string>
   </property>
  </widget>
  <widget class="QLabel" name="info_login">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Login :</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_firstName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Prénom :</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_lastName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom : </string>
   </property>
  </widget>
  <widget class="QLabel" name="info_mail">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mail :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_deconnect">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Déconnexion</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
